app/types/util/parsers/boolean-parser.test.ts

```diff
@@ -0,0 +1,148 @@
+'use strict';
```

Comment:
This is leftover from an earlier commit I had in my staging.  It's part of the Extract level parsing.